subscribed_topics:
  thermal_image_topic: /flir/raspberry/image
